homeassistant/components/bluetooth/passive_update_coordinator.py

```diff
@@ -261,12 +261,9 @@ def _async_handle_bluetooth_event(
 
         if not isinstance(new_data, PassiveBluetoothDataUpdate):
             self.last_update_success = False  # type: ignore[unreachable]
-            self.logger.error(
-                "The update_method for %s returned %s instead of a PassiveBluetoothDataUpdate",
-                self.name,
-                new_data,
+            raise ValueError(
```

Comment:
Where will the exception propagate? We're inside a bleak callback, right?

Reply:
Yes, its going to get caught by the bleak callback and log the exception.

Reply:
Ok. :+1: